src/main/java/fr/inria/coming/repairability/repairtools/JGenProg.java

```diff
@@ -63,56 +66,116 @@ public boolean filter(ChangePatternInstance instance, IRevision revision) {
 
         // see if the inserted statement occurs in the previous version of the file
```

Comment:
a fix in JGenProg should not be here but in another pull-request.

Reply:
File is deleted now.